README.md

```diff
@@ -34,3 +34,24 @@ The app should hopefully prompt you to "Reopen in Container," if you have the "D
 
 [devcontainer]: https://containers.dev
 [vscode]: https://vscode.dev
+
+Docker Instructions
+------------------------
+We will be using the Docker container to run all of your code this semester.You'll be able to write your files in a local directory and then run the code in the container. 
+
+To run the image on a machine that is NOT Apple silicon (i.e. Windows and Intel Macs):
+- Install Docker 
+- Start the Docker Daemon
+- `docker pull ghcr.io/sampsyo/cs3410-infra`
+- `docker run -it -v --rm <absolute path to "shared">:/home/student/shared gchr.io/sampsyo/cs3410-infra bash`
+
+To run the image on Apple silicon: 
+- Install Docker (`brew install --cask docker`)
+- Start Docker using `docker ps` 
+- `docker pull ghcr.io/sampsyo/cs3410-infra`
+- `docker run -it --platform linux/amd64 -v --rm <absolute path to "shared">:/home/student/shared gchr.io/sampsyo/cs3410-infra bash`
```

Comment:
Just mentioning here also that we probably don't need the `--platform` arg. I can add a note to this README that the image is multiarch!